docs/Building-Transmission.md

```diff
@@ -120,24 +120,34 @@ $ sudo make install
 ```
 
 ## On Windows ##
-For Windows XP and above there are several choices:
 
-### Cygwin environment ###
-With Cygwin https://cygwin.com/ installed, the CLI tools (transmission-remote, transmission-cli, etc.) and the daemon can be built easily.
+## Building transmission-daemon
+You need the following installed:
 
-No patches needed(\*), all the recent versions of Transmission build almost out-of-the-box (you need to install the prerequisites), and the CLI tools work better under Cygwin than those built with MinGW.
+* Visual Studio 2019 or greater (the Community Edition is sufficient - just make sure its C++ compiler, MSVC, is installed)
+* [ActivePerl](https://www.activestate.com/products/activeperl/) or [StrawberryPerl](https://strawberryperl.com)
+* [CMake](https://cmake.org/download/) (choose to add CMake to your path)
+* Possibly [Git for Windows](https://git-scm.com/download/win)
+* [Vcpkg](https://github.com/microsoft/vcpkg#quick-start-windows)
 
-(\*) At the release time of version 2.0, **libevent** is not bundled and it is also not in the Cygwin distribution (but was added later)... so you need to build it (which is as easy as ./configure, make install). To build Transmission you may need to add LDFLAGS="-L/usr/local/lib" to the configure script (LIBEVENT_LIBS does not seem to work when it comes to build all the test programs).  Additionally **libutp** needs deleting -ansi on the Makefile.
 
-With version 2.51 miniupnpc fails to build, see https://miniupnp.tuxfamily.org/forum/viewtopic.php?t#1130.
+### Install all dependencies through vcpkg
 
-Version 2.80 breaks building on Cygwin, adding this https://github.com/adaptivecomputing/torque/blob/master/src/resmom/cygwin/quota.h file to Cygwin's /usr/include/sys solves the problem.  This is no longer needed after version 2.82 (Cygwin added the header).
-
-Version 2.81 with the above workaround needs a one line patch, see ticket #5692.
-
-Version 2.82, same as 2.81.
+```
+vcpkg integrate install
+vcpkg install curl
+vcpkg install zlib
+vcpkg install openssl
+```
 
-Version 2.83, no need to add quota.h, Cygwin added it.
+### Get Transmission source and build it
+```
+git clone https://github.com/transmission/transmission
+cd transmission
+cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_TOOLCHAIN_FILE=*vcpkg root*/scripts/buildsystems/vcpkg.cmake
+./build/transmission.sln
```

Comment:
Not correct on Windows

```shell
cmake --build build --config RelWithDebInfo
```
is better here

Reply:
@mikedld is it true?

Reply:
It is true, yes. Try it for yourself.

Reply:
No this isn't what I meant
```shell
cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_TOOLCHAIN_FILE=*vcpkg root*/scripts/buildsystems/vcpkg.cmake
cmake --build build --config RelWithDebInfo
```

Reply:
Depends on user's goals. If one wants to build from CLI, sure. If one wants to build from MSVS, not so much (but still not outright incorrect). It abstracts the actual generator being used, which in this case you didn't specify explicitly so I'd say it's not guaranteed to be Visual Studio one (and so in the end the file may not exist).

It also depends on our goals with these instructions. E.g. since I'm using VSCode for development, I rarely configure the project myself but let the IDE extension do that for me, editing .vscode/settings.json from time to time to adjust the configuration options.

That's why I generally don't like this type of instructions as they're never enough, and prefer people learning about CMake on their own. But don't let this stop you if you think it is in fact beneficial. After all, I'm most probably biased.

Reply:
> No this isn't what I meant
> 
> ```shell
> cmake -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_TOOLCHAIN_FILE=*vcpkg root*/scripts/buildsystems/vcpkg.cmake
> cmake --build build --config RelWithDebInfo
> ```
 But toolchain file is required, isn`t?